frontend/src/components/NestedAccordion.tsx

```diff
@@ -62,13 +62,13 @@ const NestedAccordion: React.FC<NestedAccordionProps> = ({ sectionTitle, tasks =
     return (
         <Accordion sectionTitle={sectionTitle}>
             {tasks.map((task, index) => (
-                <div className="mt-4" key={task.id}>
-                    <AccordionInside key={task.id} {...task} />
+                <div className="mt-4" key={index}>
+                    <AccordionInside key={index} {...task} />
                 </div>
             ))}
```

Comment:
Use `key` prop only on the first element with `task.id` prop.
https://react.dev/learn/rendering-lists#keeping-list-items-in-order-with-key

Reply:
removed key in `<AccordionInside>` and used `key={task.id}` for `<div>` element. will be in the nex committed